docs/user-guide/compatibility.md

```diff
@@ -143,7 +143,7 @@ Similar to support for meta-arguments, some rules may process a dynamic block as
 
 ## Modules
 
-Resources contained within modules are ignored by default, but when the [Module Inspection](./module-inspection.md) is enabled, the arguments of module calls are inspected.
+Local modules are inspected by default. This allows you to identify issues caused by input values as well.
```

Comment:
This is confusing and suggests that local modules are inspected in the same way as recursive mode.
 
```suggestion
TFLint doesn't automatically inspect the content of modules themselves. However, by default, it will analyze their content in order to raise any issues that arise from attributes in module calls.
```

Reply:
Thanks!